token/program-2022/src/extension/confidential_transfer/processor.rs

```diff
@@ -54,7 +54,7 @@ fn process_initialize_mint(
     check_program_account(mint_info.owner)?;
     let mint_data = &mut mint_info.data.borrow_mut();
     let mut mint = StateWithExtensionsMut::<Mint>::unpack_uninitialized(mint_data)?;
-    *mint.init_extension::<ConfidentialTransferMint>()? = *confidential_transfer_mint;
+    *mint.init_extension::<ConfidentialTransferMint>(false)? = *confidential_transfer_mint;
```

Comment:
Since this is being done on an uninitialized mint, it should be ok to overwrite, no?

Reply:
Right, good catch!